spec/spec.md

```diff
@@ -50,21 +52,18 @@ data objects. This version of the specification also restricts itself to
 [Verifiable Credentials](https://www.w3.org/TR/vc-data-model/) that make use of
 [BBS+ LD-Signatures](https://w3c-ccg.github.io/ldp-bbs2020).
 
-It is anticipated that future versions of this specification will add
-support for a much broader range of messaging and data-sharing formats than
-those used in the current stable draft.
-
 ## Status of This Document
 
-WACI v0.1.0 is a _DRAFT_ specification under development by the
+[WACI DRAFT](https://identity.foundation/wallet-and-credential-interactions/)
```

Comment:
This link seems wrong to me, is it correct?

Reply:
And should it be `WACI DIDComm` rather than `WACI DRAFT` ?

Reply:
No, we are attempting to cite the "original" waci.

Reply:
I think it is important to cite the original WACI paper.
I think the 'Status of this Document' section is probably the wrong place to do so.

Reply:
Wow sorry, got my wires crossed, my mistake. I think Brent's right, this link made more sense with it pointed to the rendered spec in case the reader is reading this anywhere else (github, aggregators, who knows). We link to the WACI a few lines above in the dependency list, which makes sense.